distribution/docker/src/docker/Dockerfile

```diff
@@ -74,7 +74,12 @@ ENV TARBALL_URL https://curl.haxx.se/download/curl-\${VERSION}.tar.xz
 ENV TARBALL_PATH curl-\${VERSION}.tar.xz
 
 # Install dependencies
-RUN apk add gnupg gcc make musl-dev openssl-dev openssl-libs-static file
+RUN for iter in {1..10}; do \\
```

Comment:
I wonder since we already treat this file as a groovy template, if we could define some kind of macro to wrap some arbitrary lines in this loop with some sugar.

Reply:
I've defined a `retry_loop` helper, and I think it's an improvement. What do you think?

Reply:
Brilliant. That's much better.